libs/platform/src/lib/components/table/table.component.html

```diff
@@ -423,6 +425,17 @@
                             fd-table-cell
                             role="cell"
                         ></td>
+
+                        <td
+                            *ngIf="_isShownNavigationColumn"
+                            fd-table-cell
+                        >
+                            <i
+                                fd-table-icon
+                                [navigation]="true"
+                                class="fdp-table__navigation-indicator sap-icon--slim-arrow-left"
```

Comment:
should we handle rtl?

Reply:
It's inside the scroller container so we don't really care about what's inside, it's just to calculate the container width and the scroller width properly.